src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  box-sizing: border-box;
+  background-color: #eee;
+  border: 1px solid #000;
+  transition: 0.5s;
+  @include blockSize;
```

Comment:
It's better to separate mixins and directives from other rules with an empty line.